src/main/scala/io/sdkman/changelogs/java/AzulZuluFxMigrations.scala

```diff
@@ -2,6 +2,89 @@ package io.sdkman.changelogs.java
 
 import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
 import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{ZuluFX, Linux64, MacOSX, Version, Windows, _}
```

Comment:
I'm guessing that all these imports aren't really needed.

Reply:
I resigned as a wild card and changed it to describe only a necessary article.